src/components/order-summary/_order-summary.scss

```diff
@@ -0,0 +1,147 @@
+.bx--order-summary {
+  @include font-size('14');
+  background-color: $ui-01;
+  padding-bottom: 1.25rem;
+  width: 18rem;
+  height: 100%;
+  min-height: 100vh;
+  position: relative;
+}
+
+.bx--order-header {
+  display: flex;
+  align-items: center;
+  padding: 0 0 0 1.25rem;
+  border-bottom: 1px solid $ui-05;
+
+  .bx--dropdown {
+    flex: 1;
+    background-color: $ui-01;
+    margin-right: rem(1px);
+  }
+
+  .bx--dropdown-text {
+    max-width: rem(100px);
+  }
+
+  .bx--dropdown-link {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  .bx--dropdown[data-value=''] .bx--dropdown-text,
+   .bx--dropdown-text {
+    color: $brand-01;
+  }
+}
+
+.bx--order-header-title {
+  @include font-size('14');
+  font-weight: 700;
+  flex: 2;
+}
+
+.bx--order-list {
+  padding: 1.25rem 0;
+  margin: 0 1.25rem;
```

Comment:
Border needs to not extend to the edge, whereas in the header it extends all the way. This was the only way to accomplish this